src/main/java/io/github/thebusybiscuit/sensibletoolbox/listeners/ElevatorListener.java

```diff
@@ -65,4 +67,10 @@ public void onPlayerSneak(PlayerToggleSneakEvent event) {
         }
     }
 
+    private boolean isOnGround(@Nonnull Player player) {
+        Location location = player.getLocation();
+        Block currentBlock = location.getBlock();
+    	return !currentBlock.getRelative(BlockFace.DOWN).isEmpty() && location.getY() == location.getBlockY();
```

Comment:
Nitpicky but the indentation is a little off here :eyes:

Reply:
Haha, I noticed this aswell. Will fix in next commit